specs/lib/HalModuleParser.spec.js

```diff
@@ -547,6 +558,38 @@ describe('HalModuleParser', function () {
 				}).catch(done);
 	});
 
+	it('should work with argon softdevice (radio stack)', function (done) {
+		var filename = path.join(settings.binaries, 'argon-softdevice-6.1.1.bin');
+		var expectedPrefixInfo = {
+			moduleStartAddy: '1000',
+			moduleEndAddy: '25e24',
+			moduleFlags: 1,
```

Comment:
Would it be useful to add another file `lib/moduleFlags.js` with definitions of what the flags mean like `module.exports = { flagXYZ: 1 };`

Reply:
Yeah, I'd love for `binary-version-reader` to provide the definitions of module functions, platforms and flags. Decided against, but perhaps I'll do that at least for module functions and flags. :+1: